inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp

```diff
@@ -31,6 +31,8 @@
 
 namespace LayerTestsUtils {
 
+constexpr std::size_t maxFileNameLength = 140;
```

Comment:
Please add a comment with explanation why this limit is needed and why it is set 140.

Reply:
Commented.